### ISSUE TITLE
Problem installing leveldown as dependency 

### ISSUE THREAD

**Issue by benjaminettori** [Level/leveldown#277]
Hi

the project I'm working on has level listed as a dependency, and npm install will fail when attempting to install leveldown, with the following log. Could you please give me a sense of what is wrong here? 

Thanks in advance.

```
41616 silly install prebuild@4.1.1 C:\Users\bettori\Documents\bettori_jawn\node_modules\.staging\prebuild-ec9e36e2
41617 info lifecycle prebuild@4.1.1~install: prebuild@4.1.1
41618 silly lifecycle prebuild@4.1.1~install: no script for install, continuing
41619 silly install leveldown@1.4.5 C:\Users\bettori\Documents\bettori_jawn\node_modules\.staging\leveldown-e19cfa1d
41620 info lifecycle leveldown@1.4.5~install: leveldown@1.4.5
41621 verbose lifecycle leveldown@1.4.5~install: unsafe-perm in lifecycle true
41622 verbose lifecycle leveldown@1.4.5~install: PATH: C:\Users\bettori\AppData\Roaming\npm\node_modules\npm\bin\node-gyp-bin;C:\Users\bettori\Documents\bettori_jawn\node_modules\leveldown\node_modules\.bin;C:\Users\bettori\Documents\bettori_jawn\node_modules\.bin;C:\Program Files\nodejs;C:\Program Files\Git\cmd;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\avs\bin;C:\ProgramData\Oracle\Java\javapath;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\QuickTime\QTSystem\;C:\Program Files (x86)\Sennheiser\SoftphoneSDK\;C:\Program Files (x86)\Microsoft SQL Server\110\Tools\Binn\ManagementStudio\;C:\Program Files (x86)\Microsoft SQL Server\110\Tools\Binn\;C:\Program Files\Microsoft SQL Server\110\Tools\Binn\;C:\Program Files (x86)\Microsoft SQL Server\110\DTS\Binn\;C:\Program Files (x86)\Windows Kits\8.1\Windows Performance Toolkit\;C:\Program Files\Microsoft\Web Platform Installer\;C:\Program Files (x86)\Microsoft ASP.NET\ASP.NET Web Pages\v1.0\;C:\Program Files\Git\cmd;C:\Program Files\SourceGear\Common\DiffMerge\;C:\Program Files\nodejs\;C:\Program Files\Perforce\DVCS\;C:\Program Files\TortoiseGit\bin;C:\Users\bettori\AppData\Local\Programs\Python\Python35-32\python.exe;C:\Users\bettori\AppData\Roaming\npm;C:\Program Files\Docker Toolbox;C:\Users\bettori\AppData\Local\Programs\Python\Python35-32\python.exe;C:\Users\bettori\AppData\Local\Programs\Python\Python35-32\Scripts
41623 verbose lifecycle leveldown@1.4.5~install: CWD: C:\Users\bettori\Documents\bettori_jawn\node_modules\leveldown
41624 silly lifecycle leveldown@1.4.5~install: Args: [ '/d /s /c', 'prebuild --install' ]
41625 silly lifecycle leveldown@1.4.5~install: Returned: code: 2  signal: null
41626 info lifecycle leveldown@1.4.5~install: Failed to exec install script
41627 verbose unlock done using C:\Users\bettori\AppData\Roaming\npm-cache\_locks\staging-b76edbbb40fea3d2.lock for C:\Users\bettori\Documents\bettori_jawn\node_modules\.staging
41628 silly rollbackFailedOptional Starting
41629 silly rollbackFailedOptional Finishing
41630 silly runTopLevelLifecycles Starting
41631 silly runTopLevelLifecycles Finishing
41632 silly install printInstalled
41633 verbose stack Error: leveldown@1.4.5 install: `prebuild --install`
41633 verbose stack Exit status 2
41633 verbose stack     at EventEmitter.<anonymous> (C:\Users\bettori\AppData\Roaming\npm\node_modules\npm\lib\utils\lifecycle.js:244:16)
41633 verbose stack     at emitTwo (events.js:100:13)
41633 verbose stack     at EventEmitter.emit (events.js:185:7)
41633 verbose stack     at ChildProcess.<anonymous> (C:\Users\bettori\AppData\Roaming\npm\node_modules\npm\lib\utils\spawn.js:24:14)
41633 verbose stack     at emitTwo (events.js:100:13)
41633 verbose stack     at ChildProcess.emit (events.js:185:7)
41633 verbose stack     at maybeClose (internal/child_process.js:827:16)
41633 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
41634 verbose pkgid leveldown@1.4.5
41635 verbose cwd C:\Users\bettori\Documents\bettori_jawn
41636 error Windows_NT 6.1.7601
41637 error argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\bettori\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "install"
41638 error node v5.6.0
41639 error npm  v3.8.7
41640 error code ELIFECYCLE
41641 error leveldown@1.4.5 install: `prebuild --install`
41641 error Exit status 2
41642 error Failed at the leveldown@1.4.5 install script 'prebuild --install'.
41642 error Make sure you have the latest version of node.js and npm installed.
41642 error If you do, this is most likely a problem with the leveldown package,
41642 error not with npm itself.
41642 error Tell the author that this fails on your system:
41642 error     prebuild --install
41642 error You can get information on how to open an issue for this project with:
41642 error     npm bugs leveldown
41642 error Or if that isn't available, you can get their info via:
41642 error     npm owner ls leveldown
41642 error There is likely additional logging output above.
41643 verbose exit [ 1, true ]
```

**Comment by ralphtheninja** [comment]
@benjaminettori Could you do `npm install --verbose` and copy the output of that command into a [gist](https://gist.github.com/)?

**Comment by benjaminettori** [comment]
Hi

I solved this issue by installing Python v 2.7 instead of the v 3.5 version. One if the leveldown dependencies, node-gyp, is apparently not compatible with Python versions > 3.0.

**Comment by ralphtheninja** [comment]
> Hi
> 
> I solved this issue by installing Python v 2.7 instead of the v 3.5 version. One if the leveldown dependencies, node-gyp, is apparently not compatible with Python versions > 3.0.

:+1: Happy it worked out for you!
